modules/standard/HDFS.chpl

```diff
@@ -38,32 +334,40 @@ extern record hdfs_block_byte_map_t {
 }
 
 // Connect to HDFS
+pragma "no doc"
 extern proc hdfs_connect(out fs: c_void_ptr, path: c_string, port: int): syserr; 
 
 // Disconnect from HDFS
+pragma "no doc"
 extern proc hdfs_disconnect(fs: c_void_ptr): syserr;
 
 // Allocate an array for our locale mappings
+pragma "no doc"
 extern proc hdfs_alloc_array(n: int): char_ptr_ptr;
 
-// Create a mapping locale_name -> locale_id (need this due to hdfs and since we cant
-// pass strings inside extern records when multilocale)
+// Create a mapping locale_name -> locale_id (need this due to hdfs and since
+// we cant pass strings inside extern records when multilocale)
+pragma "no doc"
 extern proc hdfs_create_locale_mapping(ref arr: char_ptr_ptr, num: int, loc_name: c_string);
 
 // Return arr[i]
+pragma "no doc"
 extern proc hdfs_index_array(locs: qio_locale_map_ptr_t, i: int): hdfs_block_byte_map_t;
 
 // Get block owners.
 // Returns an array of hdfs_block_byte_map_t's
+pragma "no doc"
 extern proc hdfs_get_owners(f: qio_file_ptr_t, out locales: qio_locale_map_ptr_t, out num_blocks: c_int, arr: char_ptr_ptr, loc_nums:int): syserr;
 
 // ********* For multilocale ************
 // Holds a file per locale
+pragma "no doc"
 record hdfsChapelFile {
   var files: [rcDomain] file;
 }
 
 // Holds a configured HDFS filesystem per locale
+pragma "no doc"
```

Comment:
If we're documenting functions that return this type and methods on this type (i.e. hdfsChapelFileSystem.hdfsOpen()), should we also document the type itself, even if we don't reveal any of its inner workings?